src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerBuilder.cs

```diff
@@ -25,7 +28,19 @@ public class OpenIdConnectServerConfiguration {
         /// <summary>
         /// Gets the options used by the OpenID Connect server middleware.
         /// </summary>
-        public OpenIdConnectServerOptions Options { get; } = new OpenIdConnectServerOptions();
+        public OpenIdConnectServerOptions Options
+        {
+            get { return _options; }
```

Comment:
I found a few places where `Options` is supposed to be not null. That's why I put check here. Do you want me to move it there? It's not a big deal for me :)